hal/armv7a/zynq7000/types.h

```diff
@@ -52,8 +52,18 @@ typedef unsigned int size_t;
 typedef int ssize_t;
 typedef unsigned long long time_t;
 
+#define MPU_REGION_NB 16
 
 typedef struct {
+	u32 rbar;
+	u32 rasr;
+} mpu_region_t;
+
+typedef struct {
+	struct {
+		mpu_region_t region[MPU_REGION_NB];
+		u32 map[MPU_REGION_NB];
+	} mpu;
```

Comment:
Your changes should be transparent for `zynq7000`. This platform contains MMU instead of MPU.
When configuration for MMU will be introduced, this structure will be filled with appropriate fields.

Reply:
Ooops,  `git mergetool` after directory reorganization. Reverted any changes in `types.h`.